localization/english/activities/expeditions/expedition_activity_l_english.yml

```diff
@@ -62,23 +62,23 @@
 
-  expedition_site_ruins_of_valfuria_desc:0 "#D Ruins of Valfuria#!- Once a capital of a mighty Elf Empire. Large portions of the city are remarkably still intact, but be warned, something most foul is said to prowl the streets there"
+  expedition_site_ruins_of_valfuria_desc:0 "#D Ruins of Valfuria#!- Once a capital of a mighty Elf Empire. Large portions of the city are remarkably still intact, but be warned, something beastly is said to prowl the streets there."
```

Comment:
normally Id like "beastly" and it does work but this is what Im planning the evil to be like
https://wot.fandom.com/wiki/Mashadar

more of an evil spirit kind of thing. Though Im not positive yet.... I like "most foul" because in my head its something Gandalf would say, but I could see "sinister" also